test/test_nestedtensor.py

```diff
@@ -2033,8 +2033,8 @@ def test_transpose_inference_mode_interaction(self, device, dtype):
             self.assertEqual(ptT, ptT_from_ntT)
 
     @dtypes(torch.float, torch.float16, torch.double)
-    def test_view(self, device, dtype):
-        nt = random_nt(device, dtype, 4, (4, 4))
+    def test_view(self, device, dtype, layout):
+        nt = random_nt(device, dtype, 4, (4, 4), layout=layout)
```

Comment:
I think you forgot to do `@parametrize` :P

Reply:
yep this is completely wrong, I started do it and bailed lol